src/pages/EmployeeCreation/EmployeeCreationPage.jsx

```diff
@@ -0,0 +1,121 @@
+import { Avatar, Box, Button, Fab, TextField, Typography } from "@mui/material";
+import UploadBox from "./components/UploadBox";
+import PageLayout from "../../components/PageLayout";
+
+/**
+ * The `EmployeeCreationPage` function is a React component that renders a form for users to create
+ * a new employee, including fields for the employee's name, email, phone number, address, and
+ * additional information, as well as a button to upload a resume and a button to create the employee.
+ * @returns {JSX.Element} The Employee creation page component.
+ */
+
+function EmployeeCreationPage() {
+
+  const [employeeName, setEmployeeName] = useState("");
+  const [employeeEmail, setEmployeeEmail] = useState("");
+  const [employeePhoneNumber, setEmployeePhoneNumber] = useState("");
+  const [employeeAddress, setEmployeeAddress] = useState("");
+  const [employeeExtraInfo, setEmployeeExtraInfo] = useState("");
```

Comment:
Hi, { useState } is not imported, pls import, UwU 

![image](https://github.com/SE2-WS23/Skillverse-Web/assets/136272095/1a106da1-4672-44c0-b176-65890879801a)

Reply:
done UwU